mirrord-layer/tests/apps/app_node.js

```diff
@@ -0,0 +1,53 @@
+const express = require("express");
+const process = require("process");
+const app = express();
+const PORT = 80;
+var done = new Array(4).fill(false);
+
+
+function exit() {
+  server.close();
+  process.exit();
+}
+
+app.get("/", (req, res) => {
+  console.log("GET: Request completed");
+  res.send("GET");
+  done[0] = true;
```

Comment:
Consider using a dict with informative names rather than an array with indexes

Reply:
(The comments here apply to the other apps as well)

Reply:
Now using a dict or a string enum in all apps.